packages/docusaurus-plugin-content-blog/src/__tests__/frontMatter.test.ts

```diff
@@ -282,6 +282,23 @@ describe('validateBlogPostFrontMatter slug', () => {
   });
 });
 
+describe('validateBlogPostFrontMatter aliases', () => {
+  testField({
+    prefix: 'aliases',
+    validFrontMatters: [
+      {aliases: ['blog/']},
+      {aliases: ['/blog']},
+      {aliases: ['/blog/']},
+      {aliases: ['./blog']},
+      {aliases: ['../blog']},
+      {aliases: ['../../blog']},
+      {aliases: ['/api/plugins/@docusaurus/plugin-debug']},
+      {aliases: ['@site/api/asset/image.png']},
```

Comment:
I'm not sure we should accept such value 😅 


This code is unlikely to yield a great pathname

`normalizeUrl([baseUrl, routeBasePath, alias])`

and the tests do not cover these edge cases.

---

The question to me is: do we want to support relative aliases?

Is this really useful? I mean, you create a relative alias, then update the slug, and then you end up with a different alias. That might be a source of bug/confusion?

If we do want relative aliases, there's a `resolvePathname` util that could probably be applied to compute the final non-relative pathname of the alias.

But I would suggest to keep it simple for now and only accept absolute pathname aliases in validation? If someone wants a relative alias someday and has great reason to add it, we'll do an update.

Reply:
> The question to me is: do we want to support relative aliases?

Probably not - will amend